kotlin/internal/jvm/compile.bzl

```diff
@@ -346,8 +346,10 @@ def _run_merge_jdeps_action(ctx, toolchains, jdeps, outputs, deps):
         len(jdeps),
     )
 
-    # For sandboxing to work, and for this action to be deterministic, the compile jars need to be passed as inputs
-    inputs = depset(jdeps, transitive = [depset([], transitive = [dep.transitive_compile_time_jars for dep in deps])])
+    inputs = depset(jdeps)
+    if not toolchains.kt.experimental_report_unused_deps == "off":
```

Comment:
This should be safe to merge since internally the `JdepsMerge` action skips resolving the jars and their target labels when this flag is set to off.

https://github.com/bazelbuild/rules_kotlin/blob/ecfa63c4eaefcfdc01eefb51cce7af8f958cb032/src/main/kotlin/io/bazel/kotlin/builder/tasks/jvm/JdepsMerger.kt#L92